docs_user/modules/con_comparing-configuration-files-between-deployments.adoc

```diff
@@ -2,17 +2,20 @@
 
 = Comparing configuration files between deployments
 
-In order to help users to handle the configuration for the {OpenStackPreviousInstaller} and {rhos_prev_long}
-services the tool: https://github.com/openstack-k8s-operators/os-diff has been
-develop to compare the configuration files between the {OpenStackPreviousInstaller} deployment and the {rhos_long} cloud.
-Make sure Golang is installed and configured on your environment:
+To help you manage the configuration for your {OpenStackPreviousInstaller} and {rhos_prev_long} ({OpenStackShort}) services, you can compare the configuration files between your {OpenStackPreviousInstaller} deployment and the {rhos_long} cloud by using the os-diff tool.
```

Comment:
Is this procedure different from “Pulling the configuration from a director deployment” because this procedure happens BEFORE any RHOSP services have been adopted on the control plane?